common/include/pcl/pcl_tests.h

```diff
@@ -219,6 +219,52 @@ namespace pcl
         return ::testing::AssertionSuccess ();
       }
 
+
+        template<typename V>
+        ::testing::AssertionResult VectorContainsAll(const char*  expr1, const char* expr2, std::vector<V> elements, std::vector<V> v) {
```

Comment:
There is no reason to pass vectors by value (involves copy), let's do `const&` instead.